R/init.R

```diff
@@ -161,16 +163,14 @@ init <- function(data,
 
   checkmate::assert_string(title, null.ok = TRUE)
   checkmate::assert_class(data, "TealData")
-  checkmate::assert_multi_class(modules, c("teal_module", "list", "teal_modules"))
+  checkmate::check_list(modules)
+  checkmate::check_class(modules, "teal_modules")
   checkmate::assert_list(filter, min.len = 0, names = "unique")
   checkmate::assert_subset(names(filter), choices = teal.data::get_dataname(data))
   checkmate::assert_character(id, max.len = 1, any.missing = FALSE)
 
   teal.logger::log_system_info()
 
-  if (is(modules, "teal_module")) {
-    modules <- teal::modules(modules)
-  }
```

Comment:
hmmm you reverted my changes - was that intentional?

Reply:
Nope - whoops

Reply:
ok so this is weird if you look here: https://github.com/insightsengineering/teal/blob/checkmate_check_to_assert/R/init.R#L174 the changes are on this branch ??